examples/common/pigweed/rpc_services/Device.h

```diff
@@ -112,7 +112,7 @@ class Device : public pw_rpc::nanopb::Device::Service<Device>
     {
 
         uint16_t vendor_id;
-        if (DeviceLayer::ConfigurationMgr().GetVendorId(vendor_id) == CHIP_NO_ERROR)
+        if (DeviceLayer::GetDeviceInstanceInfoProvider()->GetVendorId(vendor_id) == CHIP_NO_ERROR)
```

Comment:
I don't have strong opinion what should we do with the Configuration Manager long term. It used to exist as a container for data of all kind that can be associated with the device configuration, so in between we figured out that there are some sets of data that could be extracted into modules that describe their purpose in more precise way. The example is factory data that are meant to be programmed only once, do not change and be clearly vendor specific thing, so separating it from configuration manager seemed to be good idea. 

This change only continues what was already started before, so migrating some data. Regarding changing the Configuration Manager design completely, I think it would be good to open some wider discussion with other developers.